src/HIE/Bios/Types.hs

```diff
@@ -146,12 +146,13 @@ quote opt str = ("\"" ++) .  (quote' str ++) . ("\"" ++)
 ----------------------------------------------------------------
 
 -- | The environment where this library is used.
-data Cradle = Cradle {
-  -- | The project root directory.
-    cradleRootDir    :: FilePath
-  -- | The action which needs to be executed to get the correct
-  -- command line arguments
-  , cradleOptsProg   :: CradleAction
+data Cradle = Cradle
+  { cradleRootDir    :: FilePath -- ^  The project root directory.
+  , cradleDependencies :: [FilePath] -- ^ Dependencies of a cradle.
+  -- E.g. for cabal, '*.cabal' files, for stack "stack.yaml"
```

Comment:
I'd add a note that some of the files may not exist.

Reply:
Thanks for catching it! I also documented that the FilePaths are relative to the cradleRootDir.